### ISSUE TITLE
env config does not appear to work

### ISSUE THREAD

**Issue by unphased** [jwilm/alacritty#2791]
Which operating system does the issue occur on?

Ubuntu 16.04

If on linux, are you using X11 or Wayland?

x11

When I found that the TERM env variable wasnt being set, I added another one, and that one isn't being set either. 

![image](https://user-images.githubusercontent.com/1542910/64636019-dca47580-d3ce-11e9-96d6-25d6db133910.png)

**Comment by unphased** [comment]
The problem appears to be that the `~/.alacritty.yml` file is overridden by the `~/.config/alacritty/alacritty.yml` file, which appeared magically. What is strange is that `~/.alacritty.yml` contents still take effect (for example i updated to point to a powerline successfully). Pretty strange, still investigating.

**Comment by unphased** [comment]
I think the behavior that confused me was that I installed the Powerline font at about the same time that I edited the config file that alacritty was NOT referencing, and despite this it started to use that new font file, thus leading me to believe that my setting the config file did accomplish something, even though it did not. 

So there does not appear to be an issue here. Just check the alacritty config.
